PocketKit/Tests/PocketKitTests/SearchViewModelTests.swift

```diff
@@ -1,52 +1,65 @@
 import XCTest
-import Sync
 import SharedPocketKit
 
+@testable import Sync
```

Comment:
curious, why @testable import vs regular import

Reply:
`@testable` provides unit level access, meaning that I would have access to all internal modules and methods that are also marked testable without having to individually import them.

Reply:
This makes sense. Usually, we are testing internal functions for a specific class so we import the framework that it belongs in. However, it seems that we are using the internal methods of space (which is a bit odd to me) in order to help set up and tear down our tests related to core data items.

Reply:
solved in a discussion hon slack. https://pocket.slack.com/archives/G01A1760T71/p1671037671453809